If this did not crash the test has succeeded.
